pyquil/tests/conftest.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 
 from pyquil.device import Device
-
+from pyquil.parser import parse
```

Comment:
Why this change?

Reply:
Sorry, I was in the process of trying to get tests to pass, I'll get rid of these spurious changes when tests are passing.